### PR TITLE
Registration Company Input Refactoring

### PR DESCRIPTION
**Pull request by smarcet**
* Added new option options2Show (default to 20)
* refactored query actions

### CODE REVIEW

File: src/utils/query-actions.js [hunk 1]
```diff
@@ -162,39 +276,69 @@ export const queryEvents = _.debounce(async (summitId, input, onlyPublished = fa
...
+    endpoint.addQuery('order','title,code');
+    endpoint.addQuery('page', 1);
+    endpoint.addQuery('per_page', per_page);
+
+    if(input) {
+        input = escapeFilterValue(input);
+        endpoint.addQuery('filter[]', `title@@${input},code@@${input}`);
+    }
+
+    fetch(buildAPIBaseUrl(endpoint.toString()))
```
Comment (romanetar) [inline src/utils/query-actions.js:341]: can we use _fetch in here?

File: src/utils/query-actions.js [hunk 2]
```diff
@@ -204,13 +348,32 @@ export const queryGroups = _.debounce(async (input, callback) => {
...
+    endpoint.addQuery('page', 1);
+    endpoint.addQuery('per_page', per_page);
+
+    if(input) {
+        input = escapeFilterValue(input);
+        endpoint.addQuery('filter[]', `name@@${input}`);
+    }
 
-    fetch(buildAPIBaseUrl(`/api/v1/companies?filter=${filters}&access_token=${accessToken}`))
+    fetch(buildAPIBaseUrl(endpoint.toString()))
```
Comment (romanetar) [inline src/utils/query-actions.js:376]: can we use _fetch in here?

File: src/utils/query-actions.js [hunk 3]
```diff
@@ -231,64 +397,111 @@ export const queryRegistrationCompanies = _.debounce(async (summitId, input, cal
...
+    endpoint.addQuery('order','name')
+    endpoint.addQuery('page', 1);
+    endpoint.addQuery('per_page', per_page);
+
+    if(input) {
+        input = escapeFilterValue(input);
+        endpoint.addQuery('filter[]', `name@@${input}`);
+    }
+
+    fetch(buildAPIBaseUrl(endpoint.toString()))
```
Comment (romanetar) [inline src/utils/query-actions.js:504]: can we use _fetch in here?

**Review by romanetar — COMMENTED** [review]
@smarcet please review comments

**Comment by smarcet** [comment]
@romanetar please review again much thanks!

**Review by romanetar — APPROVED** [review]
LGTM

**Pull request merged**